c/include/nnstreamer.h

```diff
@@ -141,7 +141,7 @@ typedef enum {
   ML_NNFW_TYPE_PYTORCH = 11,          /**< PyTorch (.pt). (Since 6.5) */
   ML_NNFW_TYPE_NNTR_INF = 12,         /**< Inference supported from NNTrainer, SR On-device Training Framework (Since 6.5) */
   ML_NNFW_TYPE_VD_AIFW = 13,          /**< Inference framework for Samsung Tizen TV (Since 6.5) */
-  ML_NNFW_TYPE_SRNPU = 14,            /**< SR (Samsung Research) made NPU (.tvn). (Since 6.5) */
+  ML_NNFW_TYPE_TRIXENGINE = 14,       /**< TRIV/TRIA NPU low-level driver directly (.tvn). (Since 6.5) */
```

Comment:
Please add more description (requested by VD):
```
TRIxENGINE accesses TRIV/TRIA NPU low-level drivers directly. You may need to use high-level drivers wrapping this low-level driver in some devices: e.g., AIFW.
```

Does making X lower case not allowed in Tizen? (```TRIxENGINE```)

Reply:
TRIx_ENGINE?

Reply:
OK. I will.